<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rescoreVinaDialog</class>
 <widget class="QDialog" name="rescoreVinaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoDock Vina Rescoring Parameters</string>
  </property>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>20</width>
     <height>131</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>107</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Size of the grid</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_adtFolder">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Folder containing python scripts to convert receptor and ligand files in PDBQT format.&lt;/p&gt;&lt;p&gt;Usually contained in 'MGLTools-{&lt;span style=&quot; font-style:italic;&quot;&gt;version&lt;/span&gt;}/MGLToolsPckgs/' directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_cx">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adtFolder">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>70</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Folder containing python scripts to convert receptor and ligand files in PDBQT format.&lt;/p&gt;&lt;p&gt;Usually contained in 'MGLTools-{&lt;span style=&quot; font-style:italic;&quot;&gt;version&lt;/span&gt;}/MGLToolsPckgs/' directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_exec">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vina Executable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exec">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>40</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_vinaExec">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_exec_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AutoDockTools</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>20</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>107</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Center of the grid</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_cy">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_cz">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rescoring with AutoDock Vina</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sy">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sz">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sx">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>40</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
